src/Components/Quizz/Modal.css

```diff
@@ -0,0 +1,115 @@
+
+
+.contentModalP {
+    background-color: #1f222391;
+    font-size: 1.7rem;
+    color:rgba(0, 0, 0, 0.966);
+    font-family: 'Star Jedi',sans-serif;
+    letter-spacing: 1px;
+    font-weight: 700;
+    text-transform: uppercase;
+    text-shadow:
+    1px  1px 2px gold,
+    1px -1px 2px gold,
+    -1px  1px 2px gold,
+    -1px -1px 2px gold;
+    font-stretch: extra-expanded;
+    margin-left: auto;
+    margin-right: auto;
+    left: 0;
+    right: 0;
+    position: absolute;
+    
+}
+
+
+
+.modal-wrapper {
+    position: relative;
+    background: white;
+    border: 1px solid gold;
+    box-shadow: 0 5px 8px 0 rgba(0,0,0,0.2), 0 7px 20px 0 rgba(0,0,0,0.17);
+    margin: 0 auto 0;
+    transition: all .8s;
+    width: 65%;
+}
+.palpatine {
+    position: absolute;
+    bottom: 204px;
+    left: 1px;
+    height: 325px;
+    filter: drop-shadow(5px 5px 4px white);
+}
+.modal-header {
+    background: #000000;
+    height: 40px;
+    line-height: 40px;
+    padding: 5px 20px;
+    text-align: right;
+}
+
+.modal-header h3 {
```

Comment:
Use a simple class selector like `.modal-header-title` or `.modal-title` instead of using html selectors.